### PR TITLE
Docs to make theme command launch with content

### PR DESCRIPTION
**Pull request by pvpg**
- Added documents for the theme command
- Changed composer.json details that caused a warning
- Changed the README
- Added prismic-configuration.json file
- Fixed ternary operator for home image

### CODE REVIEW

File: prismic-configuration.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "apiEndpoint": "https://your-repo-name.cdn.prismic.io/api/v2"
```
Comment (levimykel) [inline prismic-configuration.json]: Why was this added? What is it doing?

Comment (PaulinaVPG) [reply prismic-configuration.json:2]: I'm. not exactly sure if the theme looks for that key name: `apiEndpoint` of the value `your-repo-name`, but I added it this way because that's the way we do it in most cases, using a .json or .js file.

Comment (PaulinaVPG) [reply prismic-configuration.json:2]: n

Comment (PaulinaVPG) [reply prismic-configuration.json:2]: I'm. not exactly sure if the theme looks for that key name: `apiEndpoint` of the value `your-repo-name`, but I added it this way because that's how we do it in most cases, using a .json or .js file.

You'd still need to update the config.php file manually, but that's something I'll mention in the guide.

Comment (pvpg) [reply prismic-configuration.json]: I'm not exactly sure if the theme looks for the key name `apiEndpoint` of the value `your-repo-name` or both, but I added it this way because that's how we do it in most cases, using a .json or .js file, [as explained here](https://user-guides.prismic.io/en/articles/764581-how-to-create-a-theme-to-quickstart-your-projects-for-prismic)

You'd still need to update the config.php file manually, but that's something I'll mention in the guide.

Comment (levimykel) [reply prismic-configuration.json]: @PaulinaVPG Okay so you just added this file. It doesn't actually do anything then. You can remove it.

You could have always asked about how the `theme` command works ;)

It will look for `your-repo-name` and replace that in the file you specify with `--conf` when running the theme command.

So you can get rid of this and just make sure to add the `--conf` to the theme command and specify the config.php file.

Comment (pvpg) [reply prismic-configuration.json]: k, file deleted !
